### PR TITLE
Reformatted the project with black and flake8 to allow line length of…

### PR DESCRIPTION
**Pull request by bolivierjr**
… 120.

### CODE REVIEW

File: models/users.py [hunk 1]
```diff
@@ -65,7 +65,4 @@ def validate_location(self, data, **kwargs):
     def validate_format(self, data, **kwargs):
         if data == 1 or data == 2:
             return
-        raise ValidationError(
-            "Format setting must be set to 1 for "
-            "imperial or 2 for metric units first."
-        )
+        raise ValidationError("Format setting must be set to 1 for " "imperial or 2 for metric units first.")
```
Comment (tedski) [inline models/users.py]: Line collapse error

**Review by tedski — CHANGES_REQUESTED** [review]

File: models/users.py (hunk 1 above)
Comment (bolivierjr) [reply models/users.py]: Good catch! thanks.

**Comment by bolivierjr** [comment]
Fixed any unwanted trailing commas that black allowed and fixed the two line collapse errors it created.

File: plugin.py [hunk 2]
```diff
@@ -54,18 +49,11 @@ def createdb(
         except DatabaseError as exc:
             log.error(str(exc))
             irc.reply(
-                "There was an error with the database. Check logs.",
-                prefixNick=False,
+                "There was an error with the database. Check logs.", prefixNick=False,
```
Comment (bolivierjr) [inline plugin.py]: Missed a trailing comma here.

**Review by tedski — APPROVED** [review]

**Pull request merged**